backend/main.py

```diff
@@ -113,6 +113,36 @@ def new_game():
   return jsonify(firebase.put('/games', game, put_data))
 
 
+@app.route('/joingame', methods=['POST'])
+def join_game():
+  request_data = request.get_json()
+  game_to_join = request_data['gameId']
+  player_to_add_to_game = request_data['playerId']
+  name = request_data.get('name', '')
+  need_gun = request_data.get('needGun', False)
+  profile_image_url = request_data.get('profileImageUrl', '')
+  start_as_zombie = request_data.get('startAsZombie', False)
+  volunteer = request_data.get('volunteer', False)
+  be_secret_zombie = request_data.get('beSecretZombie', False)
+  user_id = request.headers['userId']
+
+  player_info = {
+    'gameId': game_to_join
+  }
+  firebase.put('/users/' + user_id + '/players', player_to_add_to_game, player_info)
```

Comment:
The spec for joinGame had made mention of the fact that the playerId did not yet exist; I believe that the decision to have both the creation of the player and the addition of the player to game in the same operation was for the sake of consistency across the database.